go/go-gin-mysql-collector/docker-compose.yml

```
@@ -0,0 +1,42 @@
+version: '3.8'
+
+services:
+  appsignal:
```

```suggestion
  appsignal-collector:
```
Let's use the same name as in other apps.